.github/workflows/build_and_test.yml

```diff
@@ -217,6 +217,16 @@ jobs:
       run: |
         python3.6 -m pip install numpy 'pyarrow<3.0.0' pandas scipy xmlrunner plotly>=4.8
         python3.6 -m pip list
+    # TODO(SPARK-35507) Move Python 3.9 installtation to the docker image
```

Comment:
Do you mean to install both Python 3.8 or 3.9?

Reply:
Oh yeah because the image is shared across other branches. If we're going to maintain the image for `master` branch only, we could remove Python 3.8. It's not urgent in any event :-).